platform-docs/docs/basic-concepts/rendering.md

```diff
@@ -3,3 +3,38 @@ sidebar_position: 8
 ---
 
 # Rendering blocks
+
+## HTML Serialization and Parsing
+
+After editing your blocks, you may choose to persist them as JSON, or serialize them to HTML.
```

Comment:
```suggestion
After editing your blocks, you may choose to persist them as JSON or serialize them to HTML.
```

Nit: The comma before "or" isn't required unless the two clauses it separates are fully independent with their own subject and verb.

Couldn't help myself either 🙂